app/controllers/api/v4/states_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V4::StatesController < APIController
+  include Api::V3::PublicApi
+
+  def sync_to_user
```

Comment:
This doesn't feel like a `sync` endpoint to me. Can we just make a simple `/states` endpoint for this?

Reply:
I went back and forth between a `sync_to_user` and `index`. Does it make sense to make this a sync endpoint (similar to the medications controller) for consistency's sake?

Reply:
Hmm, I'm kinda torn - I'd love to keep this uber simple. Wanna poke the team and see how we feel?

Reply:
Makes sense 👍🏼 . I'm also leaning towards a regular GET endpoint.